lib/src/sql/table.rs

```diff
@@ -81,6 +84,19 @@ impl Display for Table {
 	}
 }
 
+impl Serialize for Table {
+	fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+	where
+		S: serde::Serializer,
+	{
+		if is_internal_serialization() {
+			serializer.serialize_newtype_struct(TOKEN, &self.0)
+		} else {
+			serializer.serialize_some(&self.to_string())
```

Comment:
We should never get to serialising this type as a basic JSON/CBOR output, as we will always have called `compute` on this value by this time, and converted it to a primary type (Null, False, True, Number, Strand, Duration, Datetime, Uuid, Array, Object, Geometry). Therefore maybe a `serialize_none()` is better here?